src/main/java/cinema/controller/InjectController.java

```diff
@@ -0,0 +1,35 @@
+package cinema.controller;
+
+import cinema.model.Role;
+import cinema.model.User;
+import cinema.service.RoleService;
+import cinema.service.UserService;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class InjectController {
```

Comment:
```suggestion
@Component
public class DataInitializer {
```
it's not quite controller